browser/app/less-2/inc/animate/fadeInDown.less

```diff
@@ -0,0 +1,54 @@
+@-webkit-keyframes fadeInDown {
+	0% {
+		opacity: 0;
+		-webkit-transform: translateY(-20px);
```

Comment:
Need to perform a code clean on styles. We can use some of the simplified mixins to use with keyframes.